concent_api/verifier/utils.py

```diff
@@ -145,10 +145,14 @@ def delete_file(file_path: str) -> None:
 
 def generate_full_blender_output_file_name(scene_file: str, frame_number: int, output_format: str) -> str:
     base_blender_output_file_name = generate_base_blender_output_file_name(scene_file)
-    return f'{base_blender_output_file_name}{frame_number:>04}.{output_format}'
+    return f'{base_blender_output_file_name}{frame_number:>04}.{output_format.lower()}'
 
 
 def generate_base_blender_output_file_name(scene_file: str) -> str:
+    if scene_file.startswith(('\\', '/')):
```

Comment:
When can this possibly happen ?

Wouldn't it be better to validate if `compute_task_def['extra_data']['scene_file']` is correct file name in `validate_compute_task_def` function ?

Also, I have noticed now that we don't check if `scene_file` exists in `VERIFIER_STORAGE_PATH` after unpacking archives. Shouldn't we do this and if it is not instantly end verification with error ? @cameel @dybi what do you guys think ?

Reply:
1. It might be a good idea, provided we know what the correct name is ;)

2. I agree, we shold stop execution as soon as possible,

Reply:
One more thing - in case there is no `scene_file` I think we should return `VerificationResult.MISMATCH`, because it would mean provider has intentionally uploaded bad archive

Reply:
So I will do it in this PR as a part of tests of this functions.

Reply:
Done by @dybi here -> https://github.com/golemfactory/concent/pull/613#pullrequestreview-135335272

Reply:
1. Done

Reply:
@kbeker Ok, I was just curious when it can possibly happen for the `scene_file` to have bad name ?

Reply:
@rwrzesien 
> Ok, I was just curious when it can possibly happen for the `scene_file` to have bad name ?

We should use the same rules as Golem here. If Golem is able to find the scene file then we should too. If not, it's fine to treat it as a `MISMATCH` even if the provider just has a mistake in the name - he should fix his client.

Reply:
@rwrzesien after consulting with @dybi we decided to delete it because someone can just upload file from current directory and then we won't have `slash` or `backslash`